pkg/agent/apiserver/apiserver.go

```diff
@@ -52,6 +53,7 @@ func (s *agentAPIServer) Run(stopCh <-chan struct{}) error {
 
 func installHandlers(aq monitor.AgentQuerier, npq monitor.AgentNetworkPolicyInfoQuerier, s *genericapiserver.GenericAPIServer) {
 	s.Handler.NonGoRestfulMux.HandleFunc("/agentinfo", agentinfo.HandleFunc(aq))
+	s.Handler.NonGoRestfulMux.HandleFunc("/podinterface", podinterface.HandleFunc(aq))
```

Comment:
`podinterfaces` to match other apis' convention

Reply:
Updated